package.json

```diff
@@ -79,7 +80,7 @@
     "router"
   ],
   "scripts": {
-    "test": "./test/test.sh",
+    "test": "./node_modules/.bin/gulp test",
```

Comment:
This is a nice change! Thanks. Do you mind if I commit tomorrow further? I started thinking in changing this line to:
`"test": "export NODE_PATH=.:$NODE_PATH; mocha --bail --recursive"`

This way we can run `npm test`, we can get rid of `gulp-mocha` and is not necessary relative paths down there!